app/assets/javascripts/initializers/initializeReadingListIcons.js

```diff
@@ -116,6 +118,23 @@ function properButtonFromEvent(event) {
   return properElement;
 }
 
+function addHoverEffectToReadingListButton(button) {
+  if (button.classList.contains('selected')) {
+    button.addEventListener('mouseenter', readingListButtonMouseEnter);
+    button.addEventListener('mouseleave', readingListButtonMouseLeave);
+  }
+}
+
+function readingListButtonMouseHandler(event) {
+  event.preventDefault();
+  var textReplacement = this; // see .bind below, this becomes the bound text
+
+  var textSpan = event.target.getElementsByClassName('bm-success')[0];
+  textSpan.innerHTML = textReplacement;
+}
+var readingListButtonMouseEnter = readingListButtonMouseHandler.bind('UNSAVE');
```

Comment:
You can even  avoid these events completely by just using CSS. See https://codepen.io/nickytonline/pen/oRXJWm